README.md

```diff
@@ -4,33 +4,301 @@ sktm - skt manager
 sktm is an orchestrator for skt keeping track of known baselines and tested
 patches.
 
-Usage
+Prerequisites
+-------------
+
+NOTE: The instructions below are intended for Fedora.
+
+Sktm uses Jenkins to queue and execute jobs. You can install and start latest
+Jenkins like this:
+
+    sudo curl -o /etc/yum.repos.d/jenkins.repo \
+                 https://pkg.jenkins.io/redhat/jenkins.repo
+    sudo rpm --import https://pkg.jenkins.io/redhat/jenkins.io.key
+    sudo dnf install -y java jenkins
```

Comment:
For pure Fedora 27, jenkins installation fails because of missing `/dev/log` after linking it (`ln -s /run/systemd/journal/dev-log /dev/log`) it completes.

Reply:
Hmm, I have `/dev/log` present on the fresh minimal install of F28. Do we need this issue mentioned?

Reply:
That is Fedora Server 28.

Reply:
The issue wasn't present on F28, I only ran into it with F27. So depends on which version you want to base the guide on. I won't oppose not mentioning it at all since it's not related to sktm but having broken system which is not our responsibility; I only mentioned it as part of my notes in case we run into it again